### PR TITLE
fix: fix ping logging

### PR DESCRIPTION
**Pull request by DiRaiks**
- Fix logging for the Ping command.

**Review note**: suggestion to use logger function everywhere instead of default console

### CODE REVIEW

File: src/actions/ping.ts [hunk 1]
```diff
@@ -17,38 +17,38 @@ export const pingAction = async (
...
+  logger.info(`${isTTY
     ? `${colors.bold(`[${retryCount}]`)}: Requesting content at ${url}`
     : `[${retryCount}]`}: Requesting content at ${url}`,
   )
   try {
     const response = await fetch(url, { signal: AbortSignal.timeout(timeout), redirect: 'follow' })
     if (response.status === 504) {
       if (retries > 1) {
-        console.log(`🔄 Retrying in ${retryInterval / 1000} seconds...`)
+        logger.info(`🔄 Retrying in ${retryInterval / 1000} seconds...`)
```
Comment (talentlessguy) [inline src/actions/ping.ts:28]: logger adds emojis so using it here would have two emojis at the same time

File: src/actions/ping.ts [hunk 2]
```diff
@@ -17,38 +17,38 @@ export const pingAction = async (
...
-        ? logger.info((isTTY ? colors.bold(colors.green(`Online ${response.status}`)) : `Online ${response.status}`))
+        ? (isTTY ? colors.bold(colors.green(`Online ${response.status}`)) : `Online ${response.status}`)
         : response.status
       }`)
     }
   }
   catch (error) {
     if (error instanceof DOMException) {
       if (retries > 1) {
-        console.log(`⌛ Timed out. Retrying...`)
```
Comment (talentlessguy) [inline src/actions/ping.ts:47]: same here

**Review by talentlessguy — CHANGES_REQUESTED** [review]

File: src/actions/ping.ts (hunk 1 above)
Comment (DiRaiks) [reply src/actions/ping.ts]: Added another type of logger.
The idea is to have the same function always do the logging.

File: src/actions/ping.ts (hunk 2 above)
Comment (DiRaiks) [reply src/actions/ping.ts:47]: https://github.com/StauroDEV/blumen/pull/40#discussion_r1618655036

**Review by talentlessguy — APPROVED** [review]

**Pull request merged**